### PR TITLE
Edit the SDK documentation for new sample apps

### PR DESCRIPTION
**Pull request by humivo**
Edited the SDK documentation in the website to include metrics where it was missing and also link to the new sample apps.

### CODE REVIEW

File: src/docs/getting-started/java-sdk/trace-metric-auto-instr.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ description:
     instrumentations for all supported libraries and all available data exporters, providing a complete out of the box
     experience for tracing and metrics on AWS. The agent is preconfigured to generate trace IDs compatible with AWS X-Ray, which
     will also work with any other tracing system, and enables trace propagation using W3C Trace Context, B3, and X-Ray.
-path: '/docs/getting-started/java-sdk/trace-auto-instr'
+path: '/docs/getting-started/java-sdk/trace-metric-auto-instr'
```
Comment (rapphil) [inline src/docs/getting-started/java-sdk/trace-metric-auto-instr.mdx:8]: maybe rename the files to be more generic:

`/docs/getting-started/java-sdk/auto-instr`
and
`/docs/getting-started/java-sdk/manual-instr`

Comment (rapphil) [reply src/docs/getting-started/java-sdk/trace-metric-auto-instr.mdx:8]: The reason why I'm suggesting to change is because this will avoid changes in the links so that bookmarks are kept valid after we add support to logs.

File: src/docs/getting-started/go-sdk/manual-instr.mdx [hunk 2]
```diff
@@ -31,7 +31,7 @@ distributed tracing and the basics of OpenTelemetry. To learn more about getting
 ## Requirements
 Go `v1.15` or later is required to run an application using OpenTelemetry. Visit the [compatibility chart](https://github.com/open-telemetry/opentelemetry-go#compatibility) of OpenTelemetry Go SDK with different `OS`, `Go Version` and `Architecture`.
```
Comment (bryan-aguilar) [inline src/docs/getting-started/go-sdk/manual-instr.mdx:32]: This line is out of date. Please update

File: src/docs/getting-started/go-sdk/manual-instr.mdx
Comment (bryan-aguilar) [inline src/docs/getting-started/go-sdk/manual-instr.mdx:1]: How are Cloudwatch and X-ray utilized in this doc? This doc seems to cover setting up the go sdk to export traces and metrics. The only mention of CW/X-ray is that you `need an ADOT Collector` running. Can we scope this doc down a bit more? Just make it a quick guide on how to setup the otel-go sdk in your application?

File: src/docs/getting-started/go-sdk/manual-instr.mdx [hunk 3]
```diff
@@ -53,27 +56,29 @@ go get go.opentelemetry.io/otel
...
 go get go.opentelemetry.io/otel/sdk/resource
 go get go.opentelemetry.io/otel/sdk/trace
+go get go.opentelemetry.io/otel/sdk/metric
+go get go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc
 ```
 
 <SectionSeparator />
 
-## Setting up the Global Tracer
+## Setting up the Global Tracer and Meter
```
Comment (bryan-aguilar) [inline src/docs/getting-started/go-sdk/manual-instr.mdx:65]: Can we have separate sections for setting up a tracer provider and meter provider? I ask this because setting up a tracer provider requires specific configuration for x-ray id generator and context propogation. 

The meter provider if pretty generic as it does not contain any AWS specific configuations.

File: src/docs/getting-started/java-sdk.mdx [hunk 4]
```diff
@@ -17,8 +17,8 @@ backend.
 
 ## Getting Started
 
-* [Auto-Instrumentation for Traces and Metrics with the Java agent](/docs/getting-started/java-sdk/trace-auto-instr)
-* [Manual Instrumentation for Traces and Metrics with the Java SDK](/docs/getting-started/java-sdk/trace-manual-instr)
+* [Auto-Instrumentation for Traces and Metrics with the Java agent](/docs/getting-started/java-sdk/auto-instr)
+* [Manual Instrumentation for Traces and Metrics with the Java SDK](/docs/getting-started/java-sdk/manual-instr)
 
 ## Sample Code
-* [Sample Spring App using OpenTelemetry Java Auto-Instrumentation](https://catalog.us-east-1.prod.workshops.aws/workshops/31676d37-bbe9-4992-9cd1-ceae13c5116c/en-US/aws-managed-oss/adot/javawalkthrough)
```
Comment (bryan-aguilar) [inline src/docs/getting-started/java-sdk.mdx:24]: I'm not sure we want to remove this link. Do we want to instead the text that it is referring to? This points to an observability workshop. I think it belong somewhere on our website.

File: src/docs/getting-started/python-sdk/auto-instr.mdx
Comment (bryan-aguilar) [inline src/docs/getting-started/python-sdk/auto-instr.mdx:1]: I have the same feedback relating to the go docs. Do we need to reference CloudWatch/X-ray here at all? This doc seems to provide instructions on setting up python auto instrumentation for AWS services. Where it exports the data seems to be out of scope.

File: src/docs/getting-started/python-sdk/auto-instr.mdx [hunk 5]
```diff
@@ -99,17 +99,34 @@ OTEL_PYTHON_ID_GENERATOR=xray \
...
+Because there can only be one global `TracerProvider` and `MeterProvider`, manual instrumentation should not instantiate its own `TracerProvider` or `MeterProvider` if used together alongside auto-instrumentation. Given that the same `TracerProvider` and `MeterProvider` is used, custom tracing and metrics works the same way when using automatic instrumentation or manual instrumentation. For information about custom trace instrumentation, see our [docs on manual instrumentation](/docs/getting-started/python-sdk/trace-metric-manual-instr).
 
 <SectionSeparator />
 
 ## Sample Application
 
-See a [sample Flask App using OpenTelemetry Python SDK Automatic Instrumentation](https://github.com/aws-observability/aws-otel-python/tree/main/integration-test-apps/auto-instrumentation/flask).
+See a [sample Flask App using OpenTelemetry Python SDK Automatic Instrumentation](https://github.com/aws-observability/aws-otel-community/tree/master/sample-apps/python-auto-instrumentation-sample-app).
```
Comment (bryan-aguilar) [inline src/docs/getting-started/python-sdk/auto-instr.mdx:129]: This sample app does not use flask

File: src/docs/getting-started/python-sdk/manual-instr.mdx
Comment (bryan-aguilar) [inline src/docs/getting-started/python-sdk/manual-instr.mdx:1]: Same feedback in regards to CloudWatch/X-ray

File: src/docs/getting-started/python-sdk/manual-instr.mdx [hunk 6]
```diff
@@ -48,9 +48,9 @@ $ pip install opentelemetry-instrumentation-flask==0.30b1 \
 
 <SectionSeparator />
 
-## Setting up the Global Tracer
+## Setting up the Global Tracer and Meter
 
-### Sending Traces to AWS X-Ray
+### Sending Traces to AWS X-Ray and Metrics to Cloudwatch
```
Comment (bryan-aguilar) [inline src/docs/getting-started/python-sdk/manual-instr.mdx:53]: How is this setup related to cloudwatch?

File: src/docs/getting-started/go-sdk/manual-instr.mdx (hunk 2 above)
Comment (humivo) [reply src/docs/getting-started/go-sdk/manual-instr.mdx:32]: Updated this line to require v1.19 or later.

File: src/docs/getting-started/java-sdk.mdx (hunk 4 above)
Comment (humivo) [reply src/docs/getting-started/java-sdk.mdx:24]: Changed the text to refer to a workshop and then added a separate link for the Java sample app.

File: src/docs/getting-started/go-sdk/manual-instr.mdx
Comment (humivo) [reply src/docs/getting-started/go-sdk/manual-instr.mdx:1]: I have removed the lines that specified exporting to X-ray and Cloudwatch.  Made the document more general to make the backend be up to the user.

File: src/docs/getting-started/go-sdk/manual-instr.mdx (hunk 3 above)
Comment (humivo) [reply src/docs/getting-started/go-sdk/manual-instr.mdx]: Separated these sections now.

File: src/docs/getting-started/python-sdk/auto-instr.mdx
Comment (humivo) [reply src/docs/getting-started/python-sdk/auto-instr.mdx:1]: I have removed the instances where it specifies to export to Cloudwatch/X-ray

Comment (humivo) [reply src/docs/getting-started/python-sdk/auto-instr.mdx:1]: However, I have kept what we had previously which is a blurb about how the user could configure to export to X-ray

File: src/docs/getting-started/python-sdk/auto-instr.mdx (hunk 5 above)
Comment (humivo) [reply src/docs/getting-started/python-sdk/auto-instr.mdx]: I have changed the name of this link.

File: src/docs/getting-started/python-sdk/manual-instr.mdx
Comment (humivo) [reply src/docs/getting-started/python-sdk/manual-instr.mdx:1]: I have removed the instances where exporting to Cloudwatch/X-ray is specified.

File: src/docs/getting-started/python-sdk/manual-instr.mdx (hunk 6 above)
Comment (humivo) [reply src/docs/getting-started/python-sdk/manual-instr.mdx]: Have removed this from the title

**Review by bryan-aguilar — APPROVED** [review]
